llama_index/llms/openai_like.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from llama_index.bridge.pydantic import Field
+from llama_index.llms import OpenAI
+
+
+class OpenAILike(OpenAI):
+    context_window: int = Field(
+        default=512, discription="Custom context_window for custom models."
```

Comment:
I think you misspelled `description` here, and can you use a better description?  The description as is basically doesn't really tell one the impact of `context_window`. You can just reuse the `description` from `LocalAI.context_window`.

Also, why did you choose 512 as the default?  Why not import `DEFAULT_CONTEXT_WINDOW` from `llama_index.constants`?